overlays/custom-packages/labwc/default.nix

```diff
@@ -27,6 +28,11 @@
        substituteInPlace ../docs/menu.xml \
        --replace alacritty ${final.foot}/bin/foot
 
+       #frame coloring example
```

Comment:
The same applies here as well - modifying labwc per app trustworthiness color config. If you move this to `packages\labwc\default.nix`, we can drop this overlay and use that labwc package directly instead.